src/components/tabs/tabs.ts

```diff
@@ -142,6 +142,8 @@ export default class IgcTabsComponent extends EventEmitterMixin<
 
     await this.updateComplete;
 
+    this.activeTab = this.tabs.find((tab) => tab.selected);
```

Comment:
`find` will take the first match, but in the case of multiple elements marked as selected we've already established the last one wins. Then there's the matter that just a few lines below there's a `this.setSelectedTab` call that correctly picks the last selected tab or falls back the first enabled, so assigning the active tab here is redundant.